.theo/formats/stories.mdx.js

```diff
@@ -141,8 +146,14 @@ import { Meta, ColorPalette, ColorItem } from '@storybook/addon-docs/blocks';
 # ${title}
 
 \`\`\`scss
-@use 'path/to/${filename}';
-$example: ${slug}.$${kebabCase(props[0].name)}; // => ${props[0].value}
+@use 'path/to/design-tokens/${filename}';
```

Comment:
I found the `path/to/design-tokens` bit confusing. Could we either change this to `../../design-tokens` which will be the case for most components, or add a bit of documentation explaining that this is the path to the generated design tokens file relative to the file you're editing?

Reply:
Good idea 👍